### PR TITLE
Rust Gamedev Meetup 30 post

### PR DESCRIPTION
**Pull request by AngelOnFira**
I'm giving this a try, it might be a nice way to see the content from the meetup in a post that can be shared.

### CODE REVIEW

**Comment by AngelOnFira** [comment]
@ElhamAryanpur @cybersoulK @kpreid do your sections look good?

**Comment by AngelOnFira** [comment]
@scirin does this kind of post look good to you?

**Comment by ElhamAryanpur** [comment]
> @ElhamAryanpur @cybersoulK @kpreid do your sections look good?

Fire as hell 🔥thanks for having us!

**Comment by scirin** [comment]
This looks great!

**Review by scirin — APPROVED** [review]

**Comment by ElhamAryanpur** [comment]
Oh this is interesting, first time I saw the talks section!

File: content/blog/rust-gamedev-meetup-30/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+[@cybersoulK]: https://github.com/cybersoulK
+[Cybergate Discord server]: https://discord.gg/R7DkHqw7zJ
+
+### no_std: ?no_problem  | [@kpreid]
+
+![no_std: ?no_problem](no-std.png)
+
+> A brief introduction to `no_std` in Rust, and how you can use it in your
+> projects.
```
Comment (kpreid) [inline content/blog/rust-gamedev-meetup-30/index.md:63]: How about:

```
An introduction to making your Rust crates `no_std` compatible, and why you might want to do this.
```

**Review by kpreid — CHANGES_REQUESTED** [review]

File: content/blog/rust-gamedev-meetup-30/index.md (hunk 1 above)
Comment (AngelOnFira) [inline content/blog/rust-gamedev-meetup-30/index.md:63]:
```suggestion
> An introduction to making your Rust crates `no_std` compatible, and
> why you might want to do this.
```

Comment (AngelOnFira) [reply content/blog/rust-gamedev-meetup-30/index.md]: Looks good!

**Pull request merged**